docs/docs.sh

```diff
@@ -31,5 +31,8 @@ set -euo pipefail
 script_dir="$(dirname "$0")"
 
 cd "${script_dir}"
-pip3 install -r requirements.txt --upgrade --upgrade-strategy eager
+# TODO ``--ignore-installed`` now fixes an issue where pip tries to unistnall a debian installed package, but is unable to
```

Comment:
```suggestion
# TODO `--ignore-installed` now fixes an issue where pip tries to uninstall a Debian installed package, but is unable to
```

Reply:
BTW, I'd also use just one `TODO` marker. It looked a bit confusing before I read it carefully - as if there was a bunch of different TODOs here. I'd just indent it instead.

Reply:
I prefer a TODO on every line since most IDEs will highlight it a different colour than a regular comment

![one_todo](https://github.com/user-attachments/assets/e5c1a05a-b4c2-4cff-9614-5d8ac5672a40)
![todos](https://github.com/user-attachments/assets/674fc886-1d53-4f6b-a471-023ca423c81c)